Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -226,23 +227,41 @@ private void InitializeServiceLocator()
             // If the Spatial Awareness system has been selected for initialization in the Active profile, enable it in the project
             if (ActiveProfile.IsSpatialAwarenessSystemEnabled)
             {
+#if UNITY_EDITOR
+                // Setup the default spatial awareness layers in the project settings.
+                LayerExtensions.SetupLayer(31, MixedRealitySpatialAwarenessSystemProfile.SpatialAwarenessMeshesLayerName);
+                LayerExtensions.SetupLayer(30, MixedRealitySpatialAwarenessSystemProfile.SpatialAwarenessSurfacesLayerName);
```

Comment:
Per earlier comment, user definable layers are a limited quantity in Unity. The Spatial Awareness System should pre-define only one layer and allow customers to define additional layers as needed/desired.